open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/layers/base/AbstractDiscoveryDataPanel.java

```diff
@@ -932,26 +932,18 @@ private void setSearchTextFieldBackgroundColor()
      */
     protected void updateCheckboxState()
     {
-        myUpdateCheckBoxExecutor.execute(new Runnable()
+        myUpdateCheckBoxExecutor.execute(() ->

```

Comment:
Note to look into this further. I'm not sure why we need to fork a thread to fork a thread (yo dawg). We may be able to get away with just the nested thread. In any case, no need to change it here.